README.md

```diff
@@ -24,6 +24,12 @@ This project is organized into microservices. Although this repository contains
     * You will need to supply environmental variable to the microservices specifying each others' endpoints
 * Each subfolder has its own README.md with more specific information regarding building, testing, deployment, etc.
 
+There are 4 microservices in this project. They are:
+- __events__ - Add, edit, and fetch events list.
```

Comment:
Link to subdirs?

Reply:
Done https://github.com/knative-portability/large-events/pull/39/commits/b83aa083a7ca1d2deed42171697e29128db09999